### DISCUSSION TITLE
Getting `INTERNAL_INVALID_FUNCTION_REGION` on my deployment

### DISCUSSION THREAD

**Discussion by milovangudelj** [vercel/community#4504] [Help] [answered]
### Summary

Today I rebuilt one of my deployed projects and visiting the production url gives me this internal server error. I'm on a Hobby Plan so I can't even get support directly from Vercel about it. There's no other information I can get from the error reference [here](https://vercel.com/docs/errors/platform-error-codes#500:-internal_invalid_function_region).

Is there something I'm doing wrong on my side? Has anyone else got this error?

### Example

https://iowt-dashboard.vercel.app/

### Steps to Reproduce

No steps since I don't know what's causing it and how to trigger the error...

**Comment by sh11mo** [comment]
same issue here. I encountered the issue after deploying just now.

**Reply by YoojinOhDev** [reply]
just tried to run instant rollback in Vercel website, 500 error is gone.

**Comment by justoverclockl** [comment]
same issue, anyone know how to fix it? it appear only in some pages...not all. The funny thing is that the help page says:

https://vercel.com/docs/errors/platform-error-codes#500:-internal_invalid_function_region

[Troubleshoot](https://vercel.com/docs/errors/platform-error-codes#troubleshoot)
[Contact Support](https://vercel.com/help) along with the ID on the error page from your deployment.

**Reply by milovangudelj** [reply]
Function logs don't even show anything suspicious. It looks like everything should work fine but it doesn't...

**Reply by justoverclockl** [reply]
yeah happen to me also...

**Comment by YoojinOhDev** [comment]
same here in Korea. not quite sure if it has something to do with the physical region though
![Uploading image.png…]()

**Comment by justoverclockl** [comment]
i'm in italy, and i'm on hobby plan....maybe is this the reason? but yesterday was fine!

**Reply by milovangudelj** [reply]
Me too, but if it's happening in Korea too it means there's something more to it.

**Comment by YoojinOhDev** [comment]
as soon as running `instant rollback` in vercel, 500 error is gone.

**Comment by imranmd** [comment]
![image](https://github.com/vercel/community/assets/7172452/110815b2-784a-487b-a576-89e97da4d8d2)

Same issue after deploying my website: [https://www.islamicdreambook.com/dream-interpretation-search/1/](https://www.islamicdreambook.com/dream-interpretation-search/1/)

**Comment by napatwongchr** [comment]
I'm in Thailand and got this issue also. I am on hobby plan.
This is the error message that i've got.
```
500: INTERNAL_SERVER_ERROR
Code: INTERNAL_INVALID_FUNCTION_REGION
ID: sin1::fqrkn-1697532727245-f0011eafbde8
```

**Comment by milovangudelj** [comment]
It's likely an issue caused by Vercel messing up something in their services, but so far [Vercel Status](https://www.vercel-status.com/) is looking fine. Might have to wait for them to update the page tho...

**Comment by ping-maxwell** [comment]
Seeing the same issue here.
```
500: INTERNAL_SERVER_ERROR
Code: INTERNAL_INVALID_FUNCTION_REGION
ID: syd1::skscq-1697532362372-0f68b8144b92
```

**Comment by swarnava** [comment]
Hi, 

We are investigating this currently. https://www.vercel-status.com/incidents/mj79vg0ttyvw

**Reply by milovangudelj** [reply]
Thank you! :tada:

**Reply by brhx** [reply]
Thank you for jumping on this so quickly! 🥳

**Reply by ping-maxwell** [reply]
Will the [Vercel status page](https://www.vercel-status.com/) be updated?

**Reply by swarnava** [reply]
We just published status page: https://www.vercel-status.com/incidents/mj79vg0ttyvw

**Reply by milovangudelj** [reply]
It has been updated right now

**Comment by swarnava** [comment, marked as answer]
A fix has been implemented and we are monitoring the results. Affected customers need to redeploy their affected deployment(s).

**Reply by milovangudelj** [reply]
Yes, I'm waiting on the build to finish, I'll let you know how it goes. :+1:

**Reply by brhx** [reply]
I've just redeployed and it seems to be fixed! 🚀

**Reply by milovangudelj** [reply]
It's working fine again! Thank you so much for the quick response and fix!

**Reply by swarnava** [reply]
Thank you!

**Reply by ping-maxwell** [reply]
Works again!! :D

**Reply by napatwongchr** [reply]
it's works thank you !

**Comment by justoverclockl** [comment]
the issue persist for me

**Reply by milovangudelj** [reply]
Have you redeployed the affected projects?

**Reply by justoverclockl** [reply]
yes several times, now i'm trying to push something new into main branch

**Comment by justoverclockl** [comment]
yeah it works again <3
